flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/dynamic/source/DynamicKafkaSourceITTest.java

```diff
@@ -805,30 +801,31 @@ private KafkaStream getKafkaStream(
                             new ClusterMetadata(Collections.singleton(topic), properties)));
         }
     }
-
-    /** Integration test based on connector testing framework. */
-    @Nested
-    class IntegrationTests extends SourceTestSuiteBase<String> {
-        @TestSemantics
-        CheckpointingMode[] semantics = new CheckpointingMode[] {CheckpointingMode.EXACTLY_ONCE};
-
-        // Defines test environment on Flink MiniCluster
-        @SuppressWarnings("unused")
-        @TestEnv
-        MiniClusterTestEnvironment flink = new MiniClusterTestEnvironment();
-
-        @TestExternalSystem
-        DefaultContainerizedExternalSystem<TwoKafkaContainers> twoKafkas =
-                DefaultContainerizedExternalSystem.builder()
-                        .fromContainer(new TwoKafkaContainers())
-                        .build();
-
-        @SuppressWarnings("unused")
-        @TestContext
-        DynamicKafkaSourceExternalContextFactory twoClusters =
-                new DynamicKafkaSourceExternalContextFactory(
-                        twoKafkas.getContainer().getKafka0(),
-                        twoKafkas.getContainer().getKafka1(),
-                        Collections.emptyList());
-    }
+    //
+    //    /** Integration test based on connector testing framework. */
+    //    @Nested
+    //    class IntegrationTests extends SourceTestSuiteBase<String> {
+    //        @TestSemantics
+    //        CheckpointingMode[] semantics = new CheckpointingMode[]
+    // {CheckpointingMode.EXACTLY_ONCE};
+    //
+    //        // Defines test environment on Flink MiniCluster
+    //        @SuppressWarnings("unused")
+    //        @TestEnv
+    //        MiniClusterTestEnvironment flink = new MiniClusterTestEnvironment();
+    //
+    //        @TestExternalSystem
+    //        DefaultContainerizedExternalSystem<TwoKafkaContainers> twoKafkas =
+    //                DefaultContainerizedExternalSystem.builder()
+    //                        .fromContainer(new TwoKafkaContainers())
+    //                        .build();
+    //
+    //        @SuppressWarnings("unused")
+    //        @TestContext
+    //        DynamicKafkaSourceExternalContextFactory twoClusters =
+    //                new DynamicKafkaSourceExternalContextFactory(
+    //                        twoKafkas.getContainer().getKafka0(),
+    //                        twoKafkas.getContainer().getKafka1(),
+    //                        Collections.emptyList());
+    //    }
 }
```

Comment:
Why is this test commented out. I fear we are losing test coverage by this

Reply:
Thanks for pointing out this, and I've removed the comment.

This was added because there is a bug in SinkTestSuiteBase/SourceTestSuiteBase in Flink 2.0-preview1 as https://issues.apache.org/jira/browse/FLINK-36845 described, and I commented out some related cases to pass CI, but this operation was not need now.